pom.xml

```diff
@@ -33,12 +33,12 @@
         <dice-provider-artifact-id>dice-provider</dice-provider-artifact-id>
         <x509-provider-artifact-id>x509-provider</x509-provider-artifact-id>
 
-        <iot-device-client-version>1.28.0-preview-001</iot-device-client-version>
+        <iot-device-client-version>1.29.0-preview-001</iot-device-client-version>
         <iot-service-client-version>1.26.0</iot-service-client-version>
         <iot-deps-version>0.11.0</iot-deps-version>
         <provisioning-device-client-version>1.8.5</provisioning-device-client-version>
         <provisioning-service-client-version>1.7.0</provisioning-service-client-version>
-        <security-provider-version>1.4.0</security-provider-version>
+        <security-provider-version>1.3.0</security-provider-version>
```

Comment:
interesting, why are we going down a version here?

Reply:
Ah, I forgot to bump this version to match with Master

Reply:
Fixed

Reply:
I did a pull from master into preview recently that bumped this security provider version up to 1.4.0, but I forgot to do the same for the versions.json file we maintain in our internals repo. I have since updated the internals repo accordingly.